### DISCUSSION TITLE
Upgrade jump to 1.7.1

### DISCUSSION THREAD

**Discussion by mdaleo404** [opentofu/opentofu#1694] [Q&A] [answered]
Is it safe to upgrade from version 1.6.0 to 1.7.1 or do I need to upgrade to 1.6.2 first?

**Comment by ghost** [comment, marked as answer]
@mdaleo404 you can safely upgrade this way, please see the [upgrade guide](https://opentofu.org/docs/intro/upgrading/).

**Reply by mdaleo404** [reply]
Thank you @janosdebugs !
